modules/db/src/main/java/org/mule/module/db/internal/domain/connection/DefaultDbConnection.java

```diff
@@ -393,22 +404,52 @@ public int getNetworkTimeout() throws SQLException
     @Override
     protected void resolveLobs(String typeName, Object[] attributes) throws SQLException
     {
-        try (ResultSet resultSet = this.getMetaData().getAttributes(this.getCatalog(), null, typeName, null))
+        Map<Integer, List> dataTypes = getDataTypes(typeName);
+
+        for(Integer key : dataTypes.keySet())
         {
+            List dataType = dataTypes.get(key);
+            doResolveLobIn(attributes, key, (int) dataType.get(0), (String) dataType.get(1));
```

Comment:
When this is changed to resolvedDataType, this should use the declarative methods.